packages/sanity/src/desk/panes/document/DocumentPaneProvider.tsx

```diff
@@ -119,22 +114,38 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
     params.path ? pathFromString(params.path) : []
   )
   const activeViewId = params.view || (views[0] && views[0].id) || null
-  const timeline = useMemo(
-    () => historyStore.getTimeline({publishedId: documentId, enableTrace: isDev}),
-    [documentId, historyStore]
-  )
```

Comment:
`Timeline` objects are only consumed by `TimelineController`s and aren't used elsewhere throughout the studio, so this has been abstracted out